style.css

```diff
@@ -1,419 +1,420 @@
-*/
-*{
-    padding: 0;
-    margin: 0;
-}
-
-body {
-	font-family: Arial, Helvetica, sans-serif;
-	font-size: 100%
-}
-
-img {
-	max-width: 100%;
-}
-
-header img:hover {
-	opacity: 0.7;
-}
-
-ul#upperNav li:hover {
-	background-color: gray;
-}
-
-header a {
-	text-decoration: none;
-	color: white;
-}
-
-nav#navigator {
-	background-color: #2F2F2F;
-	color: white;
-}
-
-nav ul#upperNav {
-	margin-top: 0;
-	margin-bottom: 15px;
-}
-
-
-
-nav ul#upperNav li {
-	border: 2px solid white;
-	border-radius: 0.3em;
-	margin-left: 10px;
-	font-size: 0.7em;
-	padding: 8px;
-}
-
-nav ul li {
-	list-style: none;
-	padding-left: 15px;
-}
-
-section#slideshow {
-	background-color: #EAEDEE;
-}
-
-div#slides img {
-	display: block;
-	height: 480px;
-}
-
-
-div#onestop img {
-	padding: 30px;
-	display: block;
-	margin-left: 40%;
-	position: absolute;
-}
-
-div#onestop h2 {
-	margin-left: 31%;
-	margin-top: 30px;
-	font-size: 2em;
-	padding: 30px;
-	letter-spacing: 0.2em;
-}
-
-div#svg object {
-	opacity: 0.5;
-	color: white;
-	padding: 0;
-}
-
-div#svg p {
-	font-size: 0.7em;
-	text-align: center;
-}
-
-div#svg figcaption {
-	font-size: 0.8em;
-	font-style: italic;
-}
-
-
-
-object#first {
-	position: absolute;
-}
-
-object#second {
-	margin-left: 32px;
-}
-
-div#jobbuzz {
-	padding-bottom: 120px;
-}
-
-
-section#bottom {
-	clear: both;
-	background-color: #EAEDEE;
-}
-
-
-div#match img {
-	display: block;
-	margin-top: 30px;
-	margin-left: 43%;
-	position: absolute;
-}
-
-div#match h2 {
-	margin-left: 28%;
-	font-size: 2em;
-	padding: 60px;
-	letter-spacing: 0.2em;
-}
-
-footer {
-	font-size: 0.8em;
-}
+	/* styles for jobseekers.html
+		Created by Sherry, Vicent, and Dennis
+		Code review by zizuo
+	*/
+	*{
+	    padding: 0;
+	    margin: 0;
+	}
 
-div#logo img {
-	margin-bottom: 10px;
-	display: block;
-}
+	body {
+		font-family: Arial, Helvetica, sans-serif;
+		font-size: 100%
+	}
 
-div#logo p {
-	font-size: 0.8em;
-	text-align: center;
-}
-
-
-div#subForm {
-	margin: 10px;
-	font-size: 0.7em;
-	background-color: #EAEDEE;
-}
-
-div#para {
-	background-color: #D9D9DA;
-	padding: 1px;
-}
-
-div#para p {
-	margin: 15px;
-	text-align: center;
-}
-
-form#subscribe input {
-	margin: 15px;
-	padding: 10px;
-	border: 0px;
-	width: 70%;
-}
-
-
-input#submit {
-	padding: 10px;
-	background-color: #5A5A5A;
-	color: white;
-}
+	img {
+		max-width: 100%;
+	}
 
+	header img:hover {
+		opacity: 0.7;
+	}
 
-@media screen and (max-width : 768px) {
+	ul#upperNav li:hover {
+		background-color: gray;
+	}
 
-	header img {
-		width: 80%;
-		display: block;
-		margin: 0 auto;
+	header a {
+		text-decoration: none;
+		color: white;
 	}
 
-	nav ul#lowerNav {
-		padding-bottom: 20px;
-		text-align: center;
+	nav#navigator {
+		background-color: #2F2F2F;
+		color: white;
 	}
 
 	nav ul#upperNav {
-		padding: 10px;
-		margin-left: 50%;
+		margin-top: 0;
+		margin-bottom: 15px;
 	}
 
-	div#slides img {
-		margin: 0 auto;
+
+
+	nav ul#upperNav li {
+		border: 2px solid white;
```

Comment:
usually, you can group your css, put box model, position, font as together as possible, that is easy for the others to understand